### ISSUE TITLE
User Agent is not resolving SDK version

### ISSUE THREAD

**Issue by mrioan** [stormpath/stormpath-sdk-java#988]
When running ITs you see this:

user_agent: stormpath-sdk-java/${project.version} java/1.8.0_60 Mac OS X/10.11.6

**Comment by jarias** [comment]
This issue only happens executing test, cause maven doesn't replace the version in https://github.com/stormpath/stormpath-sdk-java/blob/master/impl/src/main/resources/com/stormpath/sdk/version.properties at the test stage 

@mrioan Thinks we still need to fix this but not for 1.1.0 since it doesn't affect the metrics of users using the SDK only.

**Comment by mrioan** [comment]
I investigated this for some minutes and I found no simple solution. I was investigating how we can have the variable `${project.version}` in `version.properties` be resolved before the tests start but I could not yet find a way. I am now not sure whether we should spend much more time on this. The version is being properly rendered for released code, having it also for testing will also be great but does not seem to be imperative.
